src/main/java/com/epam/page/object/generator/model/RawSearchRule.java

```diff
@@ -111,7 +111,7 @@ public boolean getAnnotation() {
             return Boolean.parseBoolean(
                     this.getValue("annotation"));
         } catch (JSONException e) {
-            return false;
+            return true;
         }
     }
```

Comment:
Kindly, add Java Doc.
it is not clear, why method `getAnnotation(){...}` returns `boolean`?
it is not clear in what cases we expect `true` and in what we expect `false`?
May be it is not a very descriptive name of the method?

Reply:
Discussed on stand up. I'll fix method name and get rid of exception